IntegrationTests/PurchaseTester/PurchaseTester/SwiftPaywall.swift

```diff
@@ -362,7 +363,7 @@ class SwiftPaywall: UIViewController {
             ])
         
         // The offerings loading indicator
-        offeringLoadingIndicator = UIActivityIndicatorView(style: .gray)
+        offeringLoadingIndicator = UIActivityIndicatorView(style: .medium)
```

Comment:
these are unrelated, but we've had these build warnings for a while and they're annoying